T21/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using static System.Console;
+
+internal class Program {
+
+   // Performs the swapping for the user's input.
+   private static void Main (string[] args) {
+      GetResponse ("Enter the first number : ", out int a);

```

Comment:
```suggestion
      GetResponse ("Enter the first number: ", out int a);

```
Always have the colon close to text